Basic-Car-Maintenance.xcodeproj/xcshareddata/xcschemes/Basic-Car-Maintenance-WidgetExtension.xcscheme

```diff
@@ -90,7 +90,6 @@
       savedToolIdentifier = ""
       useCustomWorkingDirectory = "NO"
       debugDocumentVersioning = "YES"
-      askForAppToLaunch = "Yes"
```

Comment:
Dont think this change is needed.

Reply:
@anupdsouza No it isn't but I was going to manually remove it myself, @anupdsouza Thanks for the help, but no need to go through and review others PRs without talking with me first. If you would like to be involved please DM me